src/mono/mono/mini/debugger-protocol.c

```diff
@@ -0,0 +1,313 @@
+#include <config.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#ifdef HAVE_SYS_TYPES_H
+#include <sys/types.h>
+#endif
+#ifdef HAVE_SYS_SELECT_H
+#include <sys/select.h>
+#endif
+#ifdef HAVE_SYS_SOCKET_H
+#include <sys/socket.h>
+#endif
+#ifdef HAVE_NETINET_TCP_H
+#include <netinet/tcp.h>
+#endif
+#ifdef HAVE_NETINET_IN_H
+#include <netinet/in.h>
+#endif
+#ifdef HAVE_UNISTD_H
+#include <unistd.h>
+#endif
+#include <errno.h>
+#include <glib.h>
+
+#ifdef HAVE_PTHREAD_H
+#include <pthread.h>
+#endif
+
+#ifdef HOST_WIN32
+#ifdef _MSC_VER
+#include <winsock2.h>
+#include <process.h>
+#endif
+#include <ws2tcpip.h>
+#include <windows.h>
+#endif
+
+#include <mono/utils/mono-time.h>
+#include <mono/utils/w32api.h>
```

Comment:
I think a lot of these headers are not needed in `debugger-protocol.c` - I don't see any sockets, threads, time, etc in this file.

Reply:
You are right, I'm sorry. Removed.